src/utils/url/toCanonicalURL.ts

```diff
@@ -0,0 +1,19 @@
+import { parse } from 'url';
```

Comment:
`url.parse` is [deprecated](https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost).

```suggestion
```